cmd/ddev/cmd/hostname.go

```diff
@@ -52,7 +52,7 @@ to allow ddev to modify your hosts file. If you are connected to the internet an
 		// If requested, remove all inactive host names and exit
 		if removeInactive {
 			if len(args) > 0 {
-				output.UserOut.Fatal("Invalid arguments supplied. 'ddev hostname --remove-all' accepts no arguments.")
+				util.Failed("Invalid arguments supplied. 'ddev hostname --remove-all' accepts no arguments.")
```

Comment:
When I first looked at this I thought you were changing output.UserErr.Fatal() to util.Failed() and they looked the same :) That's why I was confused in the comments. But you're properly moving from output.UserOut.Fatal to (essentially) output.UserErr.Fatal. Which makes plenty of sense.